sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -388,6 +411,47 @@ make_buffer(
 Construct a SYCL buffer instance from a pointer to a Level Zero memory allocation. Please refer to <code>make_buffer</code>
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
+</tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          image<Dimensions, AllocatorT>> &,
+    const context &Context, event AvailableEvent = {})
+```
+</td>
+<td>This API is available starting with revision 4 of this specification.
+
+Construct a SYCL image instance from a ze_image_handle_t. 
+
+Because LevelZero has no way of getting image information from an image, it must be provided.  The  <code>backend_input_t</code> is a struct type like so:
+``` C++
+struct type {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    sycl::range<Dimensions> Range;
+    ext::oneapi::level_zero::ownership Ownership{
+        ext::oneapi::level_zero::ownership::transfer};
+  };
+```
+
+Example Usage
+``` C++
+sycl::backend_input_t<BE, sycl::image<2>> ImageInteropInput{ ZeHImage, ChanOrder, ChanType, ImgRange_2D, sycl::ext::oneapi::level_zero::ownership::transfer };      
+    
+auto Image_2D  = sycl::make_image<BE, 2>(ImageInteropInput, Context);
+```
+
+ The input SYCL context <code>Context</code> must be associated with a single device, matching the device used at the prior allocation.
+The <code>Context</code> argument must be a valid SYCL context encapsulating a Level-Zero context, and the Level-Zero image must have been created on the same context.
+The <code>Ownership</code> input structure member specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details. If the behavior is "transfer" then the runtime is going to free the input Level-Zero memory allocation. 
+Synchronization rules with this API are described in Section 4.5</td>

```

Comment:
The description of `make_buffer` contains this sentence:

> Created SYCL buffer can be accessed in another contexts, not only in the provided input context.

Does that apply to `make_image` too?

Reply:
No.  There was a "multiple device/context" support that was added to the buffers, but I did not add a parallel one for the images as I wanted this first implementation to be simpler. Especially for an interim solution while we wait for SYCL Next.

Reply:
In that case, I think you need to specify this limitation.  For example, add a sentence like:

> The created SYCL image can only be accessed from kernels that are submitted to a queue using this same context.